### ISSUE TITLE
Adding Shell (.sh) support for generate-script

### ISSUE THREAD

**Issue by jschanfung** [github/gh-gei#286]
## Description

Not sure if this is a bug of feature request, it looks like generate-script can only create `.ps1` file.

Checking [GH -> GH video guide](https://www.youtube.com/watch?v=5cQkM_8n5YY) in README it seems it can support `.sh`.

Either way it would be nice to support `.sh` as `generate-script` output.

**Comment by dylan-smith** [comment]
I agree this would be a nice feature. We just recently released a version of the CLI where it has to be `.ps1`. Previously the generated script could be renamed to `.sh` or `.ps1` and it would work as either, but with the support for parallel migrations we needed to start using some shell specific features to make it work, and chose powershell (since it works on windows/linux/mac).

In the future we hope to add an option to generate script to spit out either bash or powershell, but it's not a high priority right now.
